### ISSUE TITLE
How do we sync between AppleTV 4K & iPad on same ID ?

### ISSUE THREAD

**Issue by John-foxlake** [andrefmsilva/iplaytvaio-bugs#159]
Cannot Sync AppleTV to iPad.

I loaded iPlayTV (user agent 3.3.10), on my Apple TV 4K, and configured as Xtreme server (LemoTV provider), and everything worked perfectly, so I customized all channels, and favorites, and am quite happy.

I previously had installed on my iPad (gen 9, IOS 17.4), iPlayTV version 1.1.0 (202209027.0) (user agent 1.1.0) as unsubscribed, so then I decided to subscribe to Pro.  Next, I went in and NOTHING is Syncing.  So I even attempted to force a Re-Sync on the iPad by turning off iCloud sync (No, only on this device), then turned back on (No, Reset with iCloud data), so that I don’t delete what is on from the Apple TV, and under Setup, Import, Exported all show as completed with new date/times.

I verified that both devices show as on the same Apple ID.

What can I do to fix this ?

Thank you in advance, for your support.

**Comment by John-foxlake** [comment]
Apple TV 4K model is A2843, TV OS 16.6 (20M73).

**Comment by ghost** [comment]
yeah i just subscribed to get this sync going. it takes me 40-50 min to customize my channels from my service and i can’t get this to work. wrote the dev an email, but for now i just unsubbed during free trial.
